sdk/avs/Azure.ResourceManager.Avs/tests/AvsManagementTestBase.cs

```diff
@@ -37,5 +47,29 @@ protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionReso
             var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
             return lro.Value;
         }
+
+        protected async Task<AvsPrivateCloudResource> getAvsPrivateCloudResource()
+        {
+            string subscriptionId = DefaultSubscription.Data.SubscriptionId;
+            string resourceGroupName = "avs-sdk-test";
+            string privateCloudName = "avs-sdk-test";
+            ResourceIdentifier avsPrivateCloudResourceId = AvsPrivateCloudResource.CreateResourceIdentifier(subscriptionId, resourceGroupName, privateCloudName);
+            AvsPrivateCloudResource avsPrivateCloud = Client.GetAvsPrivateCloudResource(avsPrivateCloudResourceId);
+            avsPrivateCloud = await avsPrivateCloud.GetAsync();
+            Console.WriteLine(avsPrivateCloud.Data.Name);
+            return avsPrivateCloud;
+        }
+
+        protected async Task<AvsPrivateCloudClusterResource> getAvsPrivateCloudClusterResource()
+        {
+            string subscriptionId = DefaultSubscription.Data.SubscriptionId;
+            string resourceGroupName = "avs-sdk-test";
```

Comment:
if we really want to have this hard-coded, we can put this as const class fields.

Reply:
done.